specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -1263,8 +1263,12 @@ This header contains the Bootstrap API
 /// for use by the current process. If multiple packages meet the criteria the best
 /// candidate is selected.
 ///
+/// @param majorMinorVersion major and minor version of Project Reunion's framework package, encoded as `0xMMMMNNNN` where M=Major, N=Minor (e.g. 1.2 == 0x00010002).
+/// @param versionTag version tag (if any), e.g. "prerelease".
```

Comment:
So this should be marked `_In_opt_`?

Reply:
No need. That's redundant with PCWSTR -- pointers can be null and PCWSTR data can't be changed so effectively _In_opt_